zulipterminal/api_types.py

```diff
@@ -5,6 +5,15 @@
 
 EditPropagateMode = Literal["change_one", "change_all", "change_later"]
 EmojiType = Literal["realm_emoji", "unicode_emoji", "zulip_extra_emoji"]
+# Limited flags which could be present in update_message_flag events.
+ModifiableMessageFlag = Literal["read", "starred", "collapsed"]
+MessageFlags = Literal[
```

Comment:
I'm not sure if you're done with the current iteration and wanting another review yet, but this is still plural (plus commit text).